helpers/any.js

```diff
@@ -1,7 +1,7 @@
-'use strict';
 
-const _ = require('lodash');
+'use strict';
```

Comment:
This should be the first line in the file

Reply:
whoops.  Fixed.